src/providers/incoming-data/incoming-data.ts

```diff
@@ -329,8 +329,10 @@ export class IncomingDataProvider {
     let address = parsed.address ? parsed.address.toString() : '';
     let message = parsed.message;
     let amount = parsed.amount || amountFromRedirParams;
-    if (parsed.r) this.goToPayPro(data, coin);
-    else this.goSend(address, amount, message, coin);
+    if (parsed.r) {
+      const payProUrl = this.getPayProUrl(parsed.r);
```

Comment:
We should do the same for Bitcoin Cash, shouldn't we?

Reply:
Done. Thanks